http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ApplicationConfig.scala

```diff
@@ -10,5 +11,16 @@ case class ApplicationConfig(applicationName: String,
                              leoGoogleProject: GoogleProject,
                              leoServiceAccountJsonFile: Path,
                              leoServiceAccountEmail: WorkbenchEmail,
+                             leoUrlBase: String,
```

Comment:
Seems like this should be type `URL` instead of string so we get validation at config reading time?

Reply:
Just seeing that sorry, will address before merging!